libraries/AC_Autorotation/AC_Autorotation.cpp

```diff
@@ -251,54 +321,121 @@ float AC_Autorotation::get_rpm(bool update_counter)
     return current_rpm;
 }
 
+void AC_Autorotation::initial_flare_estimate()
+{
+    //estimate hover thrust
+    float _col_hover_rad = radians(_col_min + (_col_max - _col_min)*_col_hover);
+    float b = _param_solidity*6.28f;
+    _disc_area=3.14*sq(_param_diameter*0.5f);
+    float lambda = (-(b/8.0f) + safe_sqrt((sq(b))/64.0f +((b/3.0f)*_col_hover_rad)))*0.5f;
+    float freq=_governed_rpm/60.0f;
```

Comment:
Not a big fan of this.  We are potentially assuming that the user has setup the governor (this var is set from the motors param).  
If we keep this we will need to make it very clear in the setup instructions that this param is re-used in the autorotation code even if the internal governor is not being used.

Reply:
Yeah, I agree. Retrieving the powered flight RRPM (instead of passing it from motors lib) is something I had thought of...but I wasn't sure if adding more code was the correct route to follow. I initially thought to tie this to the hover collective estimation, but hadn't look any further (yet).